human_info/src/unittest_filter.py

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env python3
+import sys
+import unittest
+import numpy as np
+import filter
+
+PKG = "human_info"
+
+
+# This class is copied from the pose_engine file.
+class Keypoint:
+    __slots__ = ['k', 'yx', 'score']
+
+    def __init__(self, k, yx, score=None):
+        self.k = k
+        self.yx = yx
+        self.score = score
+
+    def __repr__(self):
+        return 'Keypoint(<{}>, {}, {})'.format(self.k, self.yx, self.score)
+
+
+class TestHumanInfo(unittest.TestCase):
+    def setUp(self):
+        self.input = np.random.normal(0, 1, 1000)
+        self.max_input = max(self.input)
+        self.min_input = min(self.input)
+
+    def test_average_filter(self):
+        self.average_filter = filter.Average_Filter()
+        for i in self.input:
+            self.distance = i
+            self.updated_distance = self.average_filter.update(self.distance)
+        self.assertTrue(abs(self.updated_distance) < 1)
+
+    def test_outlier_rejection(self):
+        self.outlier_rejection = filter.Outlier_Rejection()
+        for i in self.input:
+            self.distance = 100 * i
+            self.updated_distance = self.outlier_rejection.update(self.distance, 'nose')
+            # It does not work to take the max of the dictionary, sinc it will then try to get the max of the keys, which are strings:
+        # self.assertTrue(abs(max(self.outlier_rejection.lst) - min(self.outlier_rejection.lst)) / 100 < 0.9 * (
```

Comment:
Is this still needed ? If you want to take the max from the dict values just do `max(self.outlier_rejection.lst.values())`.